### ISSUE TITLE
Backup NFT Images that wasn't created on Zeromint

### ISSUE THREAD

**Issue by wwitkdqsck** [zeromintapp/community#10]
Hello @treeder @r-gochain ,just wondering ..can the GoBears NFTs have a backup Images On The Cloud (Google) like the rest of NFTs minted On Zeromint Directly ?Or In General for all NFTs wasn't created on Zeromint.

Problem: In my case i use Pinata.cloud, there will be a problem with Image Not Loading if they Change their Policies for Free Users or The Site Close Down..

Or At Least For The Next Project Im Doing ( GoBulls)
Please Advice .
![BackupOnZeroMint](https://user-images.githubusercontent.com/93308499/148560634-109ee13d-c728-405b-9218-b0a3945e6727.png)

**Comment by treeder** [comment]
I noticed a lot of the images not showing up because of pinata,  I don't think there's any value in us backing up all the images though, probably best not to use pinata for your NFTs if this happens.  Maybe mint directly with us next time?

**Comment by wwitkdqsck** [comment]
Yes, doing it on Zeromint will be a better option for my next Project (GoBulls) ,will there be any Charges ?This is just a Fun Project of mine(Trying to Lower the Cost as much as possible . And i dont know anything about Creating an API.I would have done GoBears using Zeromint if the Process of Using an API is easy .

Btw,So what's the Use case of Import Collection on Zeromint ?Will there be Update for Backup Images ?

**Comment by treeder** [comment]
Import is so you can view and sell on Zeromint.

**Comment by wwitkdqsck** [comment]
If i provide a Metadata like this (For The GoBulls Project->

Will it be easy to create an API?
[GoBears_metadata-512.txt](https://github.com/zeromintapp/community/files/7830008/GoBears_metadata-512.txt)

**Comment by treeder** [comment]
Well, depends if you know how to program or not. :) 

Is this related to this issue: https://github.com/zeromintapp/community/issues/11 ?

**Comment by wwitkdqsck** [comment]
Yes  @treeder !( its related to #11 ) ,No i dont know how to program it...But if you show me some links to follow...ill try to figure it out...
